tests/integration/components/document-view-test.js

```diff
@@ -8,25 +8,10 @@ module("Integration | Component | document-view", function (hooks) {
   setupRenderingTest(hooks);
   setupMirage(hooks);
 
-  test("it renders the documents when in grid view", async function (assert) {
-    this.server.createList("document", 3);
-
-    await render(hbs`<DocumentView />`, { owner: this.engine });
-
-    await click("[data-test-toggle]");
-
-    assert.dom("[data-test-upload]").exists();
-    assert.dom("[data-test-empty]").doesNotExist();
-
-    assert.dom("[data-test-document]").exists({ count: 3 });
-    assert
-      .dom("[data-test-document]")
-      .doesNotHaveClass("document-card--selected");
-  });
```

Comment:
This can not be tested in an integration test anymore (because of the redirect). The logic is already covered in an acceptance test.